src/sys/windows/selector.rs

```diff
@@ -719,12 +722,11 @@ fn interests_to_afd_flags(interests: Interest) -> u32 {
     let mut flags = 0;
 
     if interests.is_readable() {
-        // afd::POLL_DISCONNECT for is_read_hup()
-        flags |= afd::POLL_RECEIVE | afd::POLL_ACCEPT | afd::POLL_DISCONNECT;
+        flags |= READABLE_FLAGS | READ_CLOSED_FLAGS | ERROR_FLAGS;
```

Comment:
Since both `epoll` and `kqueue` always register error flags (e.g. `EPOLLERR`) should we just do the same for Windows? /cc @piscisaureus

Reply:
It looks like that's getting done right?